gpflow/likelihoods/multilatent.py

```diff
@@ -99,24 +97,23 @@ class HeteroskedasticTFPConditional(MultiLatentTFPConditional):
     def __init__(
         self,
         distribution_class: Type[tfp.distributions.Distribution] = tfp.distributions.Normal,
-        transform: tfp.bijectors.Bijector = positive(base="exp"),
-        num_gauss_hermite_points: int = 21,
+        scale_transform: tfp.bijectors.Bijector = positive(base="exp"),
+        **kwargs,
     ):
         """
         :param distribution_class: distribution class parameterized by `loc` and `scale`
-            as first and second argument, respectivily.
-        :param transform: callable applied to the variance GP to make it positive.
+            as first and second argument, respectively.
+        :param scale_transform: callable/bijector applied to the latent
+            function modelling the scale to ensure its positivity.
             Typically, `tf.exp` or `tf.softplus`, but can be any function f: R -> R^+.
         """
 
         def conditional_distribution(Fs) -> tfp.distributions.Distribution:
             tf.debugging.assert_equal(tf.shape(Fs)[-1], 2)
             loc = Fs[..., :1]
-            scale = transform(Fs[..., 1:] / 2)
+            scale = scale_transform(Fs[..., 1:])
```

Comment:
@vdutor flagging this up as it's mildly changing behaviour - I'm assuming you had the division by 2 so that the latent GP models the log-variance, but that's only true if the transform is actually Exp, so that seemed a bit weird for the general case here. (You can always include a scale in the transform itself!)

Reply:
tbh, I personally prefer to have the latent GP to log the log-scale, even though it is a little different from the HGP paper approach. Ofc, just a matter of taste.

Reply:
Happy for it to be the log-scale.